### PR TITLE
add zh-cn multioverview.md

### PR DESCRIPTION
**Pull request by Fuchange**
I'm not sure how to translate:
*  taskbox
*  VIP
*  EIP
* intranet port of the intranet IP

Maybe the translation is not good enough. Welcome your comments

### CODE REVIEW

File: content/zh/docs/installing-on-linux/introduction/multioverview.md [hunk 1]
```diff
@@ -1,163 +1,162 @@
 ---
-title: "Multi-node Installation"
-keywords: 'Multi-node, Installation, KubeSphere'
-description: 'Multi-node Installation Overview'
+title: "多节点安装"
+keywords: 'Multi-node, Installation, KubeSpherea, 多节点, 安装'
```
Comment (Jaycean) [inline content/zh/docs/installing-on-linux/introduction/multioverview.md:3]: 'Multi-node, Installation, KubeSpherea, 多节点, 安装' => 'Multi-node, Installation, KubeSphere, 多节点, 安装'
KubeSpherea => KubeSphere

**Comment by FeynmanZhou** [comment]
@Fuchange  Please fix the conflicts and sync with the upstream master, make sure the Chinese version is consistent with the latest English content. Thanks!

**Comment by rayzhou2017** [comment]
This part is already translated.

**Pull request closed without merging**